### PR TITLE
Added ML modeling code and results

### PR DESCRIPTION
**Pull request by karanjogi**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+jupyterlab_server==2.25.3
+jupyterlab_widgets==3.0.10
+kiwisolver==1.4.4
+MarkupSafe==2.1.5
+matplotlib==3.8.0
+matplotlib-inline==0.1.6
+mistune==3.0.2
+mkl-fft==1.3.1
+mkl-random==1.2.2
+mkl-service==2.4.0
```
Comment (christang) [inline requirements.txt:64]: I was not able to install these packages and had to comment them out. The error is:

```ERROR: Could not find a version that satisfies the requirement mkl-fft (from versions: none)
ERROR: No matching distribution found for mkl-fft```

for `mkl-fft` and same issue for `mkl-random` and `mkl-service`

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+munkres==1.1.4
+nbclient==0.9.0
+nbconvert==7.16.1
+nbformat==5.9.2
+nest-asyncio==1.6.0
+notebook==7.1.1
+notebook_shim==0.2.4
+numexpr==2.8.4
+numpy==1.22.3
```
Comment (christang) [inline requirements.txt:73]: There was a dependency conflict for me for `1.22.3`.

When I unpinned the version, the one that worked for me was `1.26.4`.

**Comment by christang** [comment]
this looks great

my only comments were for the `requirements.txt` file, but once I had a version that worked for me I had no problem running `model.ipynb` all the way to the end

**Review by christang — APPROVED** [review]
you can check my comments for requirements file, but model itself looks great

File: requirements.txt (hunk 2 above)
Comment (jooho-lee-kim) [reply requirements.txt:73]: That's good.

**Comment by christang** [comment]
@karanjogi re: model set up, when you apply the scaling and pca operations, are you doing this before or after you split the data into training and test? have you checked if there is any danger of leaking information across your training and evaluation steps?

**Pull request merged**